app/controllers/content_items_controller.rb

```diff
@@ -46,6 +48,10 @@ def service_sign_in_options
     end
   end
 
+  def show_email_subscription_success_banner?
```

Comment:
When we have a helper method that's needed in the view but not the controller, we usually put it in its own helper file - see [this comment from Huw](https://github.com/alphagov/email-alert-frontend/pull/1197#discussion_r751131532) explaining it on one of my PRs.

So in this case we'd have a `app/helpers/content_items_helper.rb` file with a `ContentItemsHelper` class and this method.

What do you think?

Reply:
Thank you @alex9smith, that makes sense, I get fuzzy on what is best practice sometimes, it's good to be reminded! 

I've made the changes as suggested, let me know if that's okay.